python/hsfs/feature_view.py

```diff
@@ -125,15 +125,26 @@ def __init__(
             else {}
         )
         self._features = []
-        self._feature_view_engine: "feature_view_engine.FeatureViewEngine" = (
+        self._feature_view_engine: feature_view_engine.FeatureViewEngine = (
             feature_view_engine.FeatureViewEngine(featurestore_id)
         )
-        self._transformation_function_engine: "transformation_function_engine.TransformationFunctionEngine" = transformation_function_engine.TransformationFunctionEngine(
+        self._transformation_function_engine: transformation_function_engine.TransformationFunctionEngine = transformation_function_engine.TransformationFunctionEngine(
             featurestore_id
         )
-        self._vector_server: Optional["vector_server.VectorServer"] = None
+        self._vector_server: Optional[vector_server.VectorServer] = None
+        self._batch_scoring_server: Optional[vector_server.VectorServer] = None
         self._serving_keys = serving_keys
+        self._required_serving_keys: Set[str] = (
```

Comment:
Is the parameter `_required_serving_keys` used somewhere I could not find it being used. It look the same as `_primary_keys`